src/irmin-pack-tools/ppidx/dune

```diff
@@ -3,6 +3,6 @@
  (package irmin-pack-tools)
  (name ppidx)
  (modules ppidx)
- (libraries irmin-pack irmin-pack.unix cmdliner)
+ (libraries irmin-pack irmin-pack.unix tezos-base58 irmin-tezos cmdliner)
```

Comment:
Is `tezos-base58` needed since we are using the hash in `irmin-tezos`?

Reply:
Ho yes, my bad, I removed it from the `.opam` file and forgot the dune file, I'll fix it